### DISCUSSION TITLE
Codec use

### DISCUSSION THREAD

**Discussion by ls159753** [tayler6000/pyVoIP#97] [General]
Hi Tayler,
Congrats for the job achieved so far.
I understand that default codec is PCMU.
How can we use another codec like G721 or ALAW for example ?
Documentations says it is feasible, not sure.
Thanks,

Louis

**Comment by tayler6000** [comment]
Currently, pyVoIP supports PCMU (U-Law), PCMA (A-Law), and telephone event (currently receive only).  pyVoIP will automatically switch between PCMU or PCMA depending on what the other end supports.

The feature/Add-TLS branch adds support to further restrict this on a per call basis, so if you only want to allow PCMA you can do `phone.call("123", [PayloadType.PCMA])` but note this branch is UNSTABLE.  This will be supported in 2.0.0 however.

**Comment by ls159753** [comment]
Ok that's very clear.
Any idea of when you could add G722, if ever possible ?

Le jeu. 19 janv. 2023 à 15:36, TJ Porter ***@***.***> a
écrit :
